<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrowsSettingEditWidget</class>
 <widget class="QWidget" name="ArrowsSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="ArrowsLengthSettingEditWidget" name="lengthSettingWidget" native="true"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="ArrowsShapeSettingEditWidget" name="arrowShapeWidget" native="true"/>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>18</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>78</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidgetContainer" name="additionalSettingWidget" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QGroupBox" name="legendTextGroupBox">
             <property name="title">
              <string>Length Legend</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="ArrowsLegendSettingEditWidget" name="legendWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="colormapTab">
      <attribute name="title">
       <string>Color</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="ArrowsColorSettingEditWidget" name="colorSettingEditWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidgetContainer</class>
   <extends>QWidget</extends>
   <header>misc/qwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsLegendSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowslegendsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsColorSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowscolorsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsLengthSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowslengthsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsShapeSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowsshapesettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
